From: Tobias Heider <me@tobhe.de>
Date: Sun, 12 Feb 2023 01:23:08 +0100
Subject: debian: Adjust update-m1n1 paths for debian.

---
 update-m1n1 | 13 ++++---------
 1 file changed, 4 insertions(+), 9 deletions(-)

diff --git a/update-m1n1 b/update-m1n1
index 9dd745c..ea13945 100755
--- a/update-m1n1
+++ b/update-m1n1
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # SPDX-License-Identifier: MIT
 
 set -e
@@ -13,17 +13,12 @@ else
     . /usr/share/asahi-scripts/functions.sh
 fi
 
-: ${SOURCE:="/usr/lib/asahi-boot/"}
-: ${M1N1:="$SOURCE/m1n1.bin"}
-: ${U_BOOT:="$SOURCE/u-boot-nodtb.bin"}
+: ${M1N1:="/usr/lib/m1n1/m1n1.bin"}
+: ${U_BOOT:="/usr/lib/u-boot-asahi/u-boot-nodtb.bin"}
 : ${TARGET:="$1"}
+: ${DTBS:=$(/bin/ls -d /usr/lib/linux-image-*asahi*/apple | sort -rV | head -1)/*.dtb}
 : ${CONFIG:=/etc/m1n1.conf}
 
-if [ -z "$DTBS" ]; then
-    warn "ERROR: DTBS config unset or empty, see `/etc/default/update-m1n1`"
-    exit 1
-fi
-
 umount=false
 
 m1n1config=/run/m1n1.conf
