From: Tobias Heider <me@tobhe.de>
Date: Tue, 29 Aug 2023 02:07:39 +0200
Subject: UBUNTU: make update-grub work on ubuntu

---
 Makefile    | 4 ++--
 update-grub | 9 ++-------
 2 files changed, 4 insertions(+), 9 deletions(-)

diff --git a/Makefile b/Makefile
index 34bb217..bcc199a 100644
--- a/Makefile
+++ b/Makefile
@@ -2,8 +2,8 @@ PREFIX=/usr
 SYS_PREFIX=$(PREFIX)
 CONFIG_DIR=/etc/default
 BIN_DIR=$(PREFIX)/bin
-SCRIPTS=asahi-diagnose asahi-fwupdate update-m1n1
-ARCH_SCRIPTS=update-grub first-boot
+SCRIPTS=asahi-diagnose asahi-fwupdate update-grub update-m1n1
+ARCH_SCRIPTS=first-boot
 UNITS=first-boot.service
 MULTI_USER_WANTS=first-boot.service
 DRACUT_CONF_DIR=$(PREFIX)/lib/dracut/dracut.conf.d
diff --git a/update-grub b/update-grub
index 65421b7..dad7b6f 100755
--- a/update-grub
+++ b/update-grub
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # SPDX-License-Identifier: MIT
 
 set -e
@@ -50,16 +50,11 @@ EOF
 echo "Installing GRUB files..."
 mkdir -p "$GRUB_DIR"
 touch "$GRUB_DIR"/device.map
-dd if=/dev/zero of="$GRUB_DIR"/grubenv bs=1024 count=1 2>/dev/null
-cp -r /usr/share/grub/themes "$GRUB_DIR"
+grub-editenv "$GRUB_DIR"/grubenv create
 cp -r /usr/lib/grub/arm64-efi "$GRUB_DIR"
 rm -f "$GRUB_DIR"/arm64-efi/*.module
 mkdir -p "$GRUB_DIR"/{fonts,locale}
 cp /usr/share/grub/unicode.pf2 "$GRUB_DIR"/fonts
-for i in /usr/share/locale/*/LC_MESSAGES/grub.mo; do
-    lc="$(echo "$i" | cut -d/ -f5)"
-    cp "$i" "$GRUB_DIR"/locale/"$lc".mo
-done
 
 [[ $MODULES != *$fs* ]] && MODULES="$MODULES $fs"
 echo "Core image modules: $MODULES"
